src/housingpackage/score.py

```diff
@@ -76,28 +79,36 @@ def scoring(X_test, y_test, lin_reg, tree_reg, forest_reg, grid_search):
 
 
 def load_data(in_path):
+    """Function to load the test data"""
     prepared = pd.read_csv(in_path + "/test_X.csv")
     lables = pd.read_csv(in_path + "/test_y.csv")
     lables = lables.values.ravel()
     return prepared, lables
 
 
 def load_models(model_path):
+    """Function to load the models"""
     models = []
     for i in model_names:
         with open(model_path + "/models/" + i + ".pkl", "rb") as f:
             models.append(pickle.load(f))
+            print(models)
     return models
 
 
 def score(models, X_test, y_test):
+    """scoring function"""
     lin_scores, tree_scores, forest_scores, grid_search_scores = scoring(
         X_test, y_test, models[0], models[1], models[2], models[3]
     )
 
     return [lin_scores, tree_scores, forest_scores, grid_search_scores]
 
 
+def get_model_path():
```

Comment:
doc string missing

Reply:
This was just an additional function...I have removed it currently